TombEngine/Game/effects/lightning.cpp

```diff
@@ -26,9 +29,168 @@ namespace TEN::Effects::Lightning
 	float FloatSinCosTable[8192];
 	Vector3i LightningPos[6];
 	short LightningBuffer[1024];
-	
+		
 	std::vector<LIGHTNING_INFO> Lightning;
 
+	 TWOGUNINFO twogun[2];
+
+	 void TriggerLaserBeam(Vector3i src, Vector3i dest, short LeftRight)
+	 {
+		 TWOGUNINFO* tg;
+
+		 int j;
```

Comment:
Declare `j` in a for loop header, not separately.

Reply:
done